bootstrap.yml

```diff
@@ -70,3 +70,20 @@
         vars:
           lab_vars: "{{ (lab_name == 'scale') | ternary(scale, alias) }}"
         when: lab_name in ['scale', 'alias'] and virtual_uc != true
+
+      # use of a local time server is essential or at least a good idea for Ceph, Kerberos, Redis, 
```

Comment:
looks like Travis is failing, due to the whitespace at the end of this line.

Reply:
I found and fixed.